var/spack/repos/builtin/packages/libcircle/package.py

```diff
@@ -15,6 +15,7 @@ class Libcircle(AutotoolsPackage):
     url      = "https://github.com/hpc/libcircle/releases/download/0.2.1-rc.1/libcircle-0.2.1-rc.1.tar.gz"
```

Comment:
```suggestion
     url      = "https://github.com/hpc/libcircle/releases/download/v0.3/libcircle-0.3.0.tar.gz"
```

Reply:
Thanks @gonsie , yes, actually I tried something similar to your suggestion as the first shot.  I only added v0.3 with the version and the sha256 code.  However, then it failed to find the package when downloading.  It worked when I added the url field, as well.  My guess i that between adding the leading "v" character and also using a tag of 0.3 for a package of 0.3.0, the normal spack logic to compute the URL failed.

We could update the package-wide URL to use the new v0.3, but then my worry is that the 0.2.1 version will not download successfully.  My guess is that we'll need to use a url field on something here.  If that's true, I guess we could update the package-wide URL for the v0.3 and also update the 0.2.1 line to specify it's own url if needed.

Reply:
We could perhaps also change the github tag to v0.3.0, but I suspect that's only half the problem, with the new leading "v" being the other problem.